.travis.yml

```diff
@@ -0,0 +1,147 @@
+language: go
+
+go:
+- 1.12.8
+
+os: linux
+dist: focal
+
+git:
+  depth: false
+
+services:
+  - docker
+
+env:  
+   global:
+   - TAG=nightly
+   - RECREATE_TAGS=false
```

Comment:
can this be changed to true via UI, like in a GH action?

Reply:
> we can trigger a manual build by using the trigger build option from Travis UI

ah, I see we can. :)

Can we also trigger this from an API? today we trigger all the GH actions for releasing Che from a single action: 
* https://github.com/eclipse/che-release/blob/master/make-release.sh#L134-L187
* https://github.com/eclipse/che-release/blob/master/.github/workflows/release-orchestrate-overall.yml

Is there a way to do a single "travis orchestrator" as well?

Reply:
I'll go through this and provide more details here

Reply:
> Can we also trigger this from an API?

Yes, we can trigger jobs in Travis using API. Ref: https://docs.travis-ci.com/user/triggering-builds/


> Is there a way to do a single "travis orchestrator" as well?

We can have a single Travis orchestrator in which GitHub API will be replaced with Travis API to trigger builds. I have tried the same for che-machine-exec and che-theia.